### PR TITLE
Pronoun tagged thoughts

### PR DESCRIPTION
**Pull request by Silverstar00431**
Pronoun tags for
- General
- Deputy
- Elder
- Apprentice

### CODE REVIEW

File: resources/dicts/thoughts/alive/apprentice.json [hunk 1]
```diff
@@ -3,38 +3,38 @@
...
-            "Is saving the softest bedding for their own nest",
-            "Practices their newest battle moves",
-            "Hopes they'll learn something new today",
-            "Is feeling proud of the progress they've made",
-            "Practices their hunting on a smooth rock they found",
+            "Is saving the softest bedding for {PRONOUN/m_c/poss} own nest",
+            "Practices {PRONOUN/m_c/poss} newest battle moves",
+            "Hopes {PRONOUN/m_c/subject}'ll learn something new today",
+            "Is feeling proud of the progress {PRONOUN/cat/subject}{VERB/cat/'ve/'s} made",
+            "Practices {PRONOUN/cat/object} hunting on a smooth rock {PRONOUN/m_c/subject} found",
```
Comment (keyraven) [inline resources/dicts/thoughts/alive/apprentice.json]: The first pronoun tag in this line should be ` {PRONOUN/m_c/poss} `

File: resources/dicts/thoughts/alive/apprentice.json [hunk 2]
```diff
@@ -3,38 +3,38 @@
...
             "Feels disappointed after a missed catch",
-            "Hopes there's a juicy mouse left on the fresh-kill pile for them",
+            "Hopes there's a juicy mouse left on the fresh-kill pile for {PRONOUN/m_c/object}",
             "Tries to recite the Warrior Code from memory",
             "Feels nervous about an upcoming assessment",
-            "Practices their highest jump",
-            "Practices their farthest leap",
-            "Feels proud of themselves for doing so well in training today",
+            "Practices {PRONOUN/m_c/object} highest jump",
+            "Practices {PRONOUN/m_c/object} farthest leap",
```
Comment (keyraven) [inline resources/dicts/thoughts/alive/apprentice.json]: Thoughts on lines 19 and 20 should be tagged with ` {PRONOUN/m_c/poss}`

File: resources/dicts/thoughts/alive/apprentice.json [hunk 3]
```diff
@@ -118,9 +118,9 @@
             "Leaves fresh-kill outside the nursery for the kits",
             "Is showing r_c the proper way to do a hunter's crouch",
             "Tries not to giggle as r_c fails at a hunting crouch",
-            "Looks at r_c and wonders if they were ever that small",
+            "Looks at r_c and wonders if {PRONOUN/m_c/subject} {VERB/cat/were/was} ever that small",
             "Brought back some feathers for the kits to play with",
-            "Is showing off their new battle moves to the kits"
+            "Is showing off {PRONOUN/m_c/object} new battle moves to the kits"
```
Comment (keyraven) [inline resources/dicts/thoughts/alive/apprentice.json]: This should be `{PRONOUN/m_c/poss} `

File: resources/dicts/thoughts/alive/apprentice.json [hunk 4]
```diff
@@ -151,7 +151,7 @@
     {
         "id": "thoughtful_apprentice",
         "thoughts": [
-            "Offered to go on the dawn patrol with their mentor"
+            "Offered to go on the dawn patrol with {PRONOUN/cat/object} mentor"
```
Comment (keyraven) [inline resources/dicts/thoughts/alive/apprentice.json]: This should be `{PRONOUN/m_c/poss}`

File: resources/dicts/thoughts/alive/apprentice.json [hunk 5]
```diff
@@ -142,7 +142,7 @@
     {
         "id": "formerly_a_rogue_warrior_apprentice",
         "thoughts": [
-            "Got scolded for eating what they've caught"
+            "Got scolded for eating what {PRONOUN/cat/subject} caught"
```
Comment (keyraven) [inline resources/dicts/thoughts/alive/apprentice.json]: The cat_replacement is incorrect. This should be `{PRONOUN/m_c/subject}`

**Review by keyraven — CHANGES_REQUESTED** [review]

File: resources/dicts/thoughts/alive/apprentice.json (hunk 1 above)
Comment (Silverstar00431) [reply resources/dicts/thoughts/alive/apprentice.json]: Ah thankyou, I had a feeling I missed some.

**Pull request merged**